app/views/ui-components/v1/cards/_summary.html.erb

```diff
@@ -0,0 +1,295 @@
+<% if @hbx_enrollment.is_shop? %>
+  <% member_enrollment = @member_group.group_enrollment %>
+  <% @plan = member_enrollment.product %>
+<% else %>
+  <% @plan = @member_group %>
+<% end %>
```

Comment:
nitpick (yes I love ternaries):

```
member_enrollment = @member_group.group_enrollment if @hbx_enrollment.is_shop?
@plan = @hbx_enrollment.is_shop? ? member_enrollment.product : @member_group
```

Reply:
i think in this case a ternary makes it less readable/skimable, so i think im going to leave it as is.